ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -113,6 +113,14 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-ovn
+- always_run: false
+  as: e2e-fdp-ovn-qe
```

Comment:
i think the naming convention that matches most everything else would be `e2e-aws-ovn-fdp` adding `-qe` doesn't hurt, but also not totally neccessary unless it helps with folks understanding of what the job is for.

Reply:
Thanks. Fixed it. Lets go with the trend :)

Reply:
add aws to the job name as well. typically it's found like `e2e-aws-ovn-<whatever else you want>`

Reply:
oh, I see it's changed in the next commit. ignore this